src/components/ToDo.js

```diff
@@ -25,38 +26,78 @@ const mainTodos = [
 const ToDo = () => {
   const [todos, setTodos] = useState(mainTodos);
   const [inputText, setInputText] = useState("");
+  const [alert, setAlert] = useState({ show: false });
+  const [editingItem, setEditingItem] = useState(false);
+  const [id, setId] = useState(0);
```

Comment:
`editingItem` and `id` 
This state is duplicate. One of them should be enough for editing.